templates/Yves/Plugin/Router/RouteProviderPlugin.php.twig

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace {{ namespace }};
+
+use Spryker\Yves\Router\Plugin\RouteProvider\AbstractRouteProviderPlugin;
+use Spryker\Yves\Router\Route\RouteCollection;
+
+class {{ classname }} extends AbstractRouteProviderPlugin
+{
+    protected const {{ bundle | upper }}_INDEX = '{{ bundleDashed | lower }}-index';
+    public const ROUTE_NAME_{{ bundle | upper }} = '{{ bundleDashed | lower }}-index';
+
+    /**
+     * Specification:
+     * - Adds Routes to the RouteCollection.
```

Comment:
Can be smth like `- Adds {{ bundle }} route to the RouteCollection.` in order to make this specification item plugin related